faq/post_proof.php

```diff
@@ -82,7 +82,6 @@
                 <li><a href="#html2">HTML</a></li>
                 <li><a href="#lilypond">Lilypond</a></li>
                 <li><a href="#latex">LaTeX</a></li>
-                <li><a href="#tei">TEI</a></li>
```

Comment:
This and other changes to `faq/post_proof.php` will need to be updated in the DPOD, as well, since this document is redirected to the wiki for the production site.